test/record/single-notifierSpec.ts

```diff
@@ -124,10 +106,6 @@ describe('Single Notifier', () => {
       assert.calledOnce(callbackSpy)
       assert.calledWithExactly(callbackSpy, undefined, parsedData)
 
-      assert.calledOnce(resolveSpy)
```

Comment:
why have all of these been deleted?

Reply:
Because I simplified the `request` api to only use callbacks.

Such a weird case of passing in `{ callback }` or `{ resolve, reject }`